SignalMessaging/categories/UIColor+OWS.h

```diff
@@ -32,6 +29,37 @@ NS_ASSUME_NONNULL_BEGIN
 
 - (UIColor *)blendWithColor:(UIColor *)otherColor alpha:(CGFloat)alpha;
 
+#pragma mark - New Colors
```

Comment:
nit: Nothing is new for long, maybe "Conversation"

Reply:
Removed.  I think we'll be using these colors beyond conversations.